### PR TITLE
Radio buttons for assertion verdicts

### PR DESCRIPTION
**Pull request by stalgiag**
This PR in combination with [aria-at/#1090](https://github.com/w3c/aria-at/pull/1090) address #1045.

There are more detailed notes on the changes in [aria-at/#1090](https://github.com/w3c/aria-at/pull/1090). I attempted to make this minimally intrusive. This is ready for review but I am leaving it in draft until #1090 is merged since it is dependent on those changes.

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1207031376786697

### CODE REVIEW

File: server/scripts/import-tests/index.js [hunk 1]
```diff
@@ -47,7 +47,8 @@ Default use:
 }
 
 const ariaAtRepo = 'https://github.com/w3c/aria-at.git';
-const ariaAtDefaultBranch = 'master';
+// TODO: Change this one aria-at is updated and before merging
+const ariaAtDefaultBranch = 'assertion-verdict-radio-buttons';
```
Comment (howard-e) [inline server/scripts/import-tests/index.js:51]: Reminder to remove this before merging

File: client/components/TestRenderer/AssertionsFieldset/index.jsx [hunk 2]
```diff
@@ -28,18 +29,30 @@ const AssertionsFieldset = ({ assertions, commandIndex, assertionsHeader }) => {
...
-            <input
-              type="checkbox"
-              id={`pass-${commandIndex}-${assertionIndex}`}
-              name={`assertion-${commandIndex}-${assertionIndex}`}
-              defaultChecked={passed}
-              onClick={click}
-            />
-            {description[0]}
-          </Label>
+          <Fieldset key={`AssertionKey_${assertionIndex}`}>
```
Comment (howard-e) [inline client/components/TestRenderer/AssertionsFieldset/index.jsx:33]: Looks good to me but could the radio buttons also be aligned horizontally? Like it is in https://github.com/w3c/aria-at/pull/1090

Comment (stalgiag) [reply client/components/TestRenderer/AssertionsFieldset/index.jsx:34]: Good call! Did this here: [6079e4c](https://github.com/w3c/aria-at-app/pull/1161/commits/6079e4ceccb8a0811ade14c5519461b50383277a)

**Review by howard-e — APPROVED** [review]
Looks good to me! Changed the import branch to https://github.com/w3c/aria-at/pull/1090 to get the latest test harness and confirmed all is well locally (manually and with the simulated automation environment).

My only additional thought here is a test might be helpful here to confirm when no result is provided given what is being solved in https://github.com/w3c/aria-at-app/issues/1045.

(calling out that will need the work on https://github.com/w3c/aria-at/pull/1090 to be done again as it was merged and reverted due to sync issues with this PR not being ready)

**Comment by stalgiag** [comment]
Note to reviewers, this does pass the tests as evidenced by the testing results on this commit [4fb76de](https://github.com/w3c/aria-at-app/pull/1161/commits/4fb76de2e84db4a8172cce662c75ae7798073a10). It is only failing now because I set the `ariaAtDefaultBranch` back to 'master'

**Comment by howard-e** [comment]
See https://github.com/w3c/aria-at/pull/1093#pullrequestreview-2212656271

**Pull request merged**